### PR TITLE
chore(tf): deploy the csi-controller with terraform

### PR DESCRIPTION
**Pull request by tiagolobocastro**
chore: add variables and example file for less powered vms

Add variables to modify the hugepages, cpu, cpu mask, resource requests and
resource limits and request timeouts and poll periods.

----
chore(tf): deploy the csi-controller with terraform

---
chore: match up nats and etcd probe configs

### CODE REVIEW

File: deploy/terraform/variables.tf [hunk 1]
```diff
@@ -61,10 +76,15 @@ variable "mayastor_cpus" {
   description = "number of CPUs to use"
   default     = 2
 }
+variable "mayastor_cpu_mask" {
+  type        = string
+  description = "CPU mask"
```
Comment (gila) [inline deploy/terraform/variables.tf:81]: this should read core-list, core-mask is defined with -m...

mask: -m 0x3
or 
list: -l 0,2

Same thing the latter is perhaps what easier to deal with reason about.

**Review by gila — APPROVED** [review]

File: deploy/terraform/variables.tf (hunk 1 above)
Comment (tiagolobocastro) [reply deploy/terraform/variables.tf:81]: dohh good catch thanks.

Comment (tiagolobocastro) [reply deploy/terraform/variables.tf]: I've also added https://github.com/openebs/mayastor-control-plane/pull/107/commits/93acc1dce40ae7d98803b5e26434d7d80625389c which is probably fine for testing and speeds up installs quite a bit.

**Comment by gila** [comment]
Cool! 🙏

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**